bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html

```diff
@@ -4,15 +4,21 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-<div class="m24-t-green">
+<div class="m24-t-purple">
   <section class="m24-c-showcase">
-    <h2 class="m24-c-showcase-title">Leadership ipsum dolor sit amet</h2>
-    <p class="m24-c-showcase-subtitle">Suspendisse commodo sed erat in pellentesque. In sit amet tortor porttitor, gravida mauris tempor, pharetra erat. Nam vel justo urna.</p>
+    <div class="m24-c-showcase-text">
+      <h2 class="m24-c-showcase-title">Community love — our driving force</h2>
+      <div class="m24-c-showcase-subtitle-containter">
+        <p class="m24-c-showcase-subtitle">Mozilla isn’t just an organization—it’s a global community of passionate volunteers, fellows and contributors who have been building, protecting and shaping the internet with us since 1998.</p>
+        <p class="m24-c-showcase-subtitle">From writing code and spotting bugs to advocating for privacy and keeping the internet open for everyone, they’re the backbone of everything we do, and we couldn’t do this without them. And guess what? You can join the team too!</p>
+      </div>
+    </div>
+
     <div class="m24-c-showcase-media">
       <img src="{{ static('img/m24/todo-1-1.svg') }}" alt="" height="500" width="500">
     </div>
-    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.about.leadership.index') }}" class="m24-c-cta" data-cta-text="Leadership">Leadership</a></p>
+    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.about.leadership.index') }}" class="m24-c-cta" data-cta-text="Leadership">Join the movement</a></p>
```

Comment:
**Issue/Suggestion:* the link needs updating here to point to `/contribute/`?

```suggestion
    <p class="m24-c-showcase-cta"><a href="{{ url('mozorg.contribute') }}" class="m24-c-cta" data-cta-text="Join the movement">Join the movement</a></p>
```

Also, should we rename this include from `leadership.html` to `community.html`?

Reply:
Updated to use `community` here as file name and the link